acracensor/handlers/querycapture_handler.go

```diff
@@ -156,7 +156,12 @@ func (handler *QueryCaptureHandler) CheckQuery(query string) error {
 	queryInfo.IsForbidden = false
 	handler.Queries = append(handler.Queries, *queryInfo)
 
-	handler.logChannel <- *queryInfo
+	select {
```

Comment:
wow good, I didn't know about such construction to avoid block of write to channel )